packages/eslint-plugin-xstate/src/rules/avoid-context-spread.test.ts

```diff
@@ -0,0 +1,48 @@
+import { ESLintUtils } from "@typescript-eslint/utils";
+import outdent from "outdent";
+import rule from "./avoid-context-spread";
+
+const ruleTester = new ESLintUtils.RuleTester({
+  parser: "@typescript-eslint/parser",
+});
+
+ruleTester.run("avoid-context-spread", rule, {
+  valid: [
+    {
+      code: outdent`
```

Comment:
Using `outdent` allows the code blocks to be displayed correctly in test output. Not sure if there's a better way to handle that but without it, the indentation was a mess.